bib/cmd/bootc-image-builder/build_type.go

```diff
@@ -39,3 +44,21 @@ func NewBuildType(imageTypes []string) (BuildType, error) {
 
 	return supportedImageTypes[imageTypes[0]], nil
 }
+
+// allImageTypesString returns a comma-separated list of supported types
+func allImageTypesString() string {
+	keys := make([]string, 0, len(supportedImageTypes))
+	for k := range supportedImageTypes {
+		keys = append(keys, k)
+	}
+	sort.Strings(keys)
+
+	r := ""
```

Comment:
This loop can be just strings.Join(), see https://pkg.go.dev/strings#Join

Reply:
Thanks, yep!  Originally I was looping over the map values, but then realized it'd be a good idea to sort.

Reply:
This PR got rolled into https://github.com/osbuild/bootc-image-builder/pull/268